app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str,
```

Comment:
It will looks better if you start all parameters from new line
https://github.com/mate-academy/py-distance-class/pull/930#discussion_r1449982199